demisto_sdk/commands/content_graph/objects/conf_json.py

```diff
@@ -78,6 +78,7 @@ class ConfJSON(StrictBaseModel):
     tests: List[Test]
     skipped_tests: Dict[str, str]
     skipped_integrations: Dict[str, str]
+    native_nightly_packs: List[str]
```

Comment:
to fix the failing unit tests, make it an optional field (at least for now, maybe we'll make it required later)